pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandler.java

```diff
@@ -327,7 +327,9 @@ Map<String, List<Operation>> computeOperations(SegmentDirectory.Reader segmentRe
         }
       }
     }
-
+    if (!columnOperationsMap.isEmpty()) {
+      LOGGER.info("Need to apply columnOperations: {} for forward index", columnOperationsMap);
```

Comment:
Not introduced in this PR, but seems we are not logging table name or segment name, which can make it hard to debug

Reply:
Can be addressed separately

Reply:
refined logs a bit more